bin/make/test_integration_phpcr.mk

```diff
@@ -0,0 +1,10 @@
+
+test_integration_phpcr:
+	@if [ "${CONSOLE}" = "" ]; then echo "Console executable missing"; exit 1; fi
+	@echo
+	@echo '+++ create PHPCR +++'
```

Comment:
+ database ?

Reply:
PHPCR stands for "PHP content repository", so this seems correct to me. we could say "reset PHPCR" instead, maybe.